lib/bugsnag/api/client/releases.rb

```diff
@@ -0,0 +1,38 @@
+module Bugsnag
+  module Api
+    class Client
+      # Methods for the Releases API
+      module Releases
+        # List the Releases in a Project
+        #
+        # @option release_stage [String] Only Releases with this release stage will be returned
+        # @option base [String] Only Releases created before this time will be returned
+        # @option sort [String] How to sort the results, one of: timestamp, percent_of_sessions
+        # @option offset [Number] The pagination offset
+        # @option per_page [Number] The number of results to return per page
+        # @return [Array<Sawyer::Resource>] List of Events for the specified Error
+        # @see https://bugsnagapiv2.docs.apiary.io/#reference/projects/releases/list-releases-on-a-project
+        def releases(project_id, options = {})
+          get "projects/#{project_id}/releases", options
+        end
+
+        # View a single Release
+        #
+        # @see https://bugsnagapiv2.docs.apiary.io/#reference/projects/releases/view-a-release
+        def release(project_id, release_id, options = {})
```

Comment:
Should this accept options? There doesn't seem to be any.

Reply:
There are no options on the API endpoint, but we allow passing some headers in the `options` parameter so every method has it:

https://github.com/bugsnag/bugsnag-api-ruby/blob/b0c5e15bfcc01dd757eaa05997315263f2f1c11c/lib/bugsnag/api/client.rb#L211-L224

Reply:
ah right! makes sense.